APE6.rst

```diff
@@ -559,6 +582,9 @@ a double-quote within a string, hence the double-double quotes.
   "{""b"":[2.5,null]}"
   true
 
+In this subtype, the ``null`` marker is decoded by JSON as the language-specific
```

Comment:
This sentence sounds a bit confusing to me (maybe it's a language-specific
use of the term "JSON" to mean "the JSON parsing library").  I'd suggest
instead something like:

> The ``null`` token is the JSON representation of a null value.  JSON parsers will typically decode this to some language-specific value, for instance ``None`` in Python.